app/index.html

```diff
@@ -2,15 +2,32 @@
 <html>
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  <link type="text/css" href="font-awesome/css/font-awesome.min.css" rel="stylesheet"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link type="text/css" href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Cabin:400,600" rel="stylesheet"> 
   <link type="text/css" href="styles/main.css" rel="stylesheet" media="screen">
   <title>Headers</title>
 </head>
 <body>
   <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   <div class="wrapper">
   	<div class="header header__first"></div>
-  	<div class="header header__second"></div>
+  	<div class="header header-second">
+      <div class="header-second-content">
+         <label for="header-second__bars" class="header-second__bars fa fa-bars" aria-hidden="true">
+          </label>
+           <input class="header-second__check" id="header-second__bars" type="checkbox">
```

Comment:
why do you need here this input? As I can see in psd, it is just a button for menu dropdown or something like that
And please don't use `input` without or outside of `form` tag because it won't work

Reply:
I use this to open the menu on the mobile. Tag `checkbox` - display none